HW1/P6/P6.py

```diff
@@ -0,0 +1,81 @@
+import findspark
+findspark.init()
+
+import pyspark
+sc = pyspark.SparkContext()
+
+import numpy as np
+import random
+
+def counter(list):
+    
+    d={}
+    for i in list:       
+        d[i] = list.count(i)
+    return d
```

Comment:
I dont understand what counter(list) is doing. I would suggest adding some more comments to your code so that others can provide better feedback.